borrowing_service/tests/test_borrowing_model.py

```diff
@@ -0,0 +1,75 @@
+from datetime import timezone
+from unittest import TestCase
+
+from rest_framework import status
+
+from books.models import Book
+from borrowing_service.models import Borrowing
+from rest_framework.test import APIClient
+
+from borrowing_service.serializers import BorrowingDetailSerializer, BorrowingSerializer
+
+
+class BorrowingModelTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(username="Testuser")
```

Comment:
you should use get_user_model() along with .create_user method instead of User.objects.create